### ISSUE TITLE
SyntaxError: Unable to parse JSON string

### ISSUE THREAD

**Issue by thbar** [handsontable/handsontable#3045]
When running jasmine specs offline via [teaspoon](https://github.com/modeset/teaspoon), and if I happen to use handsontable and jasmine fixtures, I get this error all the time. This apparently does not make the specs fail, this is just in the output for some reason.

I realize that my report is pretty blurry at the moment, sorry about that. It could be something super specific, yet I felt until I'm able to provide a clear reproduction, this is still be worth sharing.

Here is the error I get (one error for each spec involving handsontable):

```
SyntaxError: Unable to parse JSON string
  # handsontable.full.js:20069 -- _generate
  # handsontable.full.js:20037 -- _generate
  # handsontable.full.js:19999 -- generate
  # handsontable.full.js:19921
  # handsontable.full.js:19932
```

Sorry I cannot provide more information at this point, I hope to be back with more.

**Comment by AMBudnik** [comment]
Hi @thbar 
How is your progress in this matter?
I hope that you already solved this one out but if not, please leave a comment and we'll reopen this issue.

**Comment by budnix** [comment]
This bug still occurs in the latest version. Patch was merged into develop and waiting for release. It should be open until patch will be released.

**Comment by jansiegel** [comment]
Released in `0.25.0`.
